### ISSUE TITLE
ISSUIE ON RUNNING INSTAGRAM BOT

### ISSUE THREAD

**Issue by valentine-sys** [redianmarku/instagram-follower-scraper#22]
I have done all there requirements you told me to do but when I run python3 run.py , I get there followers I want to scrape alright and amount followers also alright... But there is an error that stops it,, please help in solving it. Please See there image,
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/66960299/226837280-c1c8f262-991a-45ed-b170-5500781e7c69.png">

**Comment by redianmarku** [comment]
Are you sure that you have created the .env file and put username and password of your instagram. Or you can just rename the .env.example into .env and put your credentials.

**Comment by valentine-sys** [comment]
Yes I added my username and password in there .env file,, and also renamed  there file  to .env

**Comment by redianmarku** [comment]
what version of python are you using? Make sure to use python3 in your commands and also pip3 when you install requirements

**Comment by ajstahovec** [comment]
Can confirm; I am getting the same issue. 
I am using python3 and pip3, and both are up-to-date. My .env file is just named "untitled.env"—I assume that isn't an issue.
<img width="1081" alt="Screenshot 2023-03-23 at 11 50 13 PM" src="https://user-images.githubusercontent.com/35577027/227381766-a39d65bc-dbb8-41d2-b219-5077a35551dd.png">

**Comment by redianmarku** [comment]
naming "untitled.env" is an issue, .env is an environment file and it should always be named like that. We use environment file to hide sensitive information like password in this case. If you are having problems getting infos about your account from .env you can edit the code and replace it directly in the code.

**Comment by ajstahovec** [comment]
What should I name the .env file then? It can't just have an extension for a name--not on MacOS at least.

**Comment by redianmarku** [comment]
please use a code editor like VScode because you cant edit it directly on mac

**Comment by ajstahovec** [comment]
Okay edited it on VSCode, still running into the same error message as before.
<img width="1086" alt="Screenshot 2023-03-25 at 8 05 51 PM" src="https://user-images.githubusercontent.com/35577027/227739098-65d0c7bf-9847-4926-bb4f-143c812fd7e4.png">

**Comment by labhayl** [comment]
If you are using it for personal use, to avoid this error you can save your username and password in **run.py** file instead of .env at line 12 and 13 respectively.
<img width="191" alt="image" src="https://user-images.githubusercontent.com/60417905/228335740-238d2e54-5548-4bfe-93e3-c29bc0443357.png">
